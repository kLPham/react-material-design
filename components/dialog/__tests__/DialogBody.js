import React from 'react';
import { shallow } from 'enzyme';
import DialogBody from '../dialogBody';

describe('<DialogBody />', () => {
    it('should render', () => {
        const dialogBody = shallow(
            <DialogBody>
                Let Google help apps determine location. This means sending anonymous location data to Google, even when no apps are running.
            </DialogBody>,
        );
        expect(dialogBody).toMatchSnapshot();
    });
});
