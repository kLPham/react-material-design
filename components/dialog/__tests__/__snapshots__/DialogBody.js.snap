// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DialogBody /> should render 1`] = `
<section
  className="mdc-dialog__body"
  id="my-mdc-dialog-description"
>
  Let Google help apps determine location. This means sending anonymous location data to Google, even when no apps are running.
</section>
`;
